lib/bolt/config.rb

```diff
@@ -170,6 +170,13 @@ def update_from_cli(options)
           self[:transports][transport][key] = options[key] if options[key]
         end
       end
+
+      if options[:sudo_password] && self[:transports][:ssh][:run_as].nil?
+        logger = Logger.new(self[:log_destination])
+        logger.warn("--sudo-password only provides privilege escalation "\
```

Comment:
this message isn't super clear to me.
something like:
'--sudo-password will not be used without specifying a user to escalate to with --run-as'

Reply:
Fixed